### ISSUE TITLE
Can it improve the bend resolution?

### ISSUE THREAD

**Issue by Wenzy--** [Nagitch/MidiJack#9]
Hi,Nagitch.Your addons is great.I am using roli seaboard.
But is there any way I can improve the bend data's resolution?I have tried to map the bend data to visualize a keyboard.The picture below it's the minimum unit when I slide the key left and right.Compare to the Equator,it looks rough.In Euqator,i can see there's a KeyTracking under the interface and the slide movement can be more smooth.

and I found when I press the middle of the key.The bend output  always 0.007874016 or -0.007874016.That can't be zero.So if I map the data directly.This's always an offset exist.

[img]https://i.imgur.com/dxn3x4c.png[/img]

**Comment by Nagitch** [comment]
@Wenzy-- 
Thanks for feedback. you have exciting instrument and project.
roli seaboard can make pitch bend from sliding key after pressing down it, right?

Pitch bend is -8192 to +8191 value from specification of MIDI, so MidiMaster.GetBend() should steps roughly 1/8192=0.0001220703125, from minimal neighbor on Equator,
and just on Equator, should be 0.

I'll investigate it.

**Comment by Wenzy--** [comment]
Yes.Except the bend data,everything works fine in this addons now.I get the data from the bend function in  the DelegateTester.cs you wrote.I also try to use the MidiMaster.GetBend()  directly but has the same output~

**Comment by Nagitch** [comment]
ok, thanks! sorry for patience and take time... I'm handling this in off-time.
